express-api/src/services/properties/propertiesServices.ts

```diff
@@ -0,0 +1,42 @@
+import { AppDataSource } from '@/appDataSource';
+import { Building } from '@/typeorm/Entities/Building';
+import { Parcel } from '@/typeorm/Entities/Parcel';
+
+const propertiesFuzzySearch = async (keyword: string, limit?: number) => {
```

Comment:
Maybe we should give this a default of undefined. In testing, it required me to fill the second argument with undefined manually.